static/app/views/alerts/rules/metric/details/metricChartOption.tsx

```diff
@@ -355,6 +403,72 @@ export function getMetricAlertChartOption({
       });
   }
 
+  if (anomalies) {
+    const anomalyBlocks: any[] = [];
+    let start;
+    let end;
+    anomalies
+      .filter(anomalyts => {
+        const ts = new Date(anomalyts.timestamp).getTime();
+        return firstPoint < ts && ts < lastPoint;
+      })
+      .forEach(anomalyts => {
+        const {anomaly, timestamp} = anomalyts;
+
+        if (
+          [AnomalyType.high, AnomalyType.low].indexOf(anomaly.anomaly_type as string) > -1
```

Comment:
nit using includes instead of -1 indexof
```suggestion
          [AnomalyType.high, AnomalyType.low].includes(anomaly.anomaly_type as string)
```